app/src/main/res/values/strings.xml

```diff
@@ -1,13 +1,27 @@
 <resources>
     <string name="app_name">Structure_Android</string>
+
+    <!-- Dialog -->
+    <string name="agree">Agree</string>
+    <string name="disagree">Disagree</string>
+    <string name="allow">Allow</string>
+    <string name="deny">Deny</string>
+    <string name="dont_ask_again">Don\'t ask again</string>
+    <string name="more_info">More Info</string>
+    <string name="choose">Choose</string>
+    <string name="clear">Clear</string>
+
+    <!-- Common -->
+    <string name="error">Error</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.error` appears to be unused
 Unused resources make applications larger and slow down builds.